plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/ProjectSpecRunner.kt

```diff
@@ -178,7 +178,6 @@ private fun writeSettingFile(path: Path) {
 
     dependencyResolutionManagement {
         repositories {
-            mavenLocal()
```

Comment:
Having this made Kotlin/JS compilation fail for some reason https://youtrack.jetbrains.com/issue/KT-61760, fortunately we don't need it.